ci/qa/phpmd

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)/../../
+
+./vendor/bin/phpmd src text ci/qa/phpmd.xml --exclude */Tests/*
```

Comment:
Please add a linefeed to this file. As a convention all files end with an empty line. :man_shrugging: